### PR TITLE
Merge revision into develop

### PR DESCRIPTION
**Pull request by omichel**
Fixed conflicts in:
```
src/webots/nodes/WbIndexedFaceSet.cpp
src/webots/nodes/WbSpotLight.cpp 
```

### CODE REVIEW

**Comment by omichel** [comment]
👍

**Comment by DavidMansolino** [comment]
test me

File: src/webots/nodes/WbIndexedFaceSet.cpp [hunk 1]
```diff
@@ -21,6 +21,7 @@
 #include "WbMFInt.hpp"
 #include "WbMatter.hpp"
 #include "WbNodeUtilities.hpp"
+#include "WbOdeGeomData.hpp"
 #include "WbNormal.hpp"
```
Comment (DavidMansolino) [inline src/webots/nodes/WbIndexedFaceSet.cpp]: This is not compliant with our clang format convention. (`WbNormal.hpp` and `WbOdeGeomData.hpp` should be switched).

**Comment by omichel** [comment]
test me

**Comment by DavidMansolino** [comment]
Replaced by #532

**Pull request closed without merging**